lib/sandbox.js

```diff
@@ -119,6 +120,16 @@ function init ( ) {
     }
   };
 
+  function displayBg (bg) {
+    if (Number(bg) == 39) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/758176621/issues/source?bid=2449627&fileBranchId=2449700#l124)